### DISCUSSION TITLE
Would it be possible to hide or change these on the home screen

### DISCUSSION THREAD

**Discussion by GasCan2024** [ChristopherKlay/StadiaEnhanced#78] [Ideas & Requests]
Hi,

Would it be possible to hide or change these on the home screen:

1. Hide the column with the 3 boxes next to the main window and then extend the main window so that its centered
2. Hide the "Get Stadia Pro" box at the top.
3. Stop the autoplay of screen shots when you hover over games in your library.

Thanks for a great extension.

**Comment by ChristopherKlay** [comment]
> Hide the column with the 3 boxes next to the main window and then extend the main window so that its centered

Definitely possible. I just have to find a good solution on how to handle the size change, because just removing the side elements and giving the entire space to the "last played" title would result in the screenshot below, where the tile is higher than the screen due to scaling. It's added to the to-do list, because honestly.. i never used those tiles once so far.
![](https://i.imgur.com/rcNYTZo.png)

> Hide the "Get Stadia Pro" box at the top.

On it!

> Stop the autoplay of screen shots when you hover over games in your library.

This one is most likely not easily done, because the used classes are loaded via Stadias own JS. I will look into it however, maybe i can just overwrite the style changes.

**Comment by GasCan2024** [comment]
Thanks for looking into this and also would it be possible to have an on/off toogle in the menu for the "Pro" field, the one to the right of "Store".
